src/config/2019.json

```
@@ -1056,6 +1056,16 @@
       "website": "https://data-seo.com",
       "github": "voltek62"
     },
+    "hakaCode": {
+      "name": "William Sandres",
+      "teams": [
+        "Translators",
```

Lowercase T and no comma:

```suggestion
        "translators"
```